### PR TITLE
[CI:DOCS] troubleshooting.md: mention "podman unshare chown 0:0 path"

### PR DESCRIPTION
**Pull request by eriksjolund**
* Mention the command "podman unshare chown 0:0 dir1/a"
  that changes file ownership to the regular user's UID and GID on
  the host.

Signed-off-by: Erik Sjölund <erik.sjolund@gmail.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
/approve
@TomSweeneyRedHat PTAL

File: troubleshooting.md [hunk 1]
```diff
@@ -991,12 +991,15 @@ less: dir1/a: Permission denied
 
 #### Solution
 
-If you want to read or remove such a file, you can do so by entering a user namespace.
-Instead of running commands such as `less dir1/a` or `rm dir1/a`, you would need to
-prepend the command-line with `podman unshare`, i.e.,
-`podman unshare less dir1/a` or `podman unshare rm dir1/a`. To be able to use Bash
-features, such as variable expansion and globbing, you need to wrap the command with
-`bash -c`, e.g. `podman unshare bash -c 'ls $HOME/dir1/a*'`.
+If you want to read, chown or remove such a file, you can do so by entering a user
```
Comment (TomSweeneyRedHat) [inline troubleshooting.md:994]:
```suggestion
If you want to read, chown or remove such a file, you can do so by entering a user
```

**Comment by TomSweeneyRedHat** [comment]
couple nits, and some on older stuff, but would appreciate the cleanup.  Thanks @eriksjolund !

**Review by flouthoc — APPROVED** [review]
LGTM

**Comment by flouthoc** [comment]
@eriksjolund Could you please squash the commits.

**Comment by eriksjolund** [comment]
> @eriksjolund Could you please squash the commits.

@flouthoc fixed

**Pull request merged**